students/adam_strong/lesson03/slicing.py

```diff
@@ -0,0 +1,54 @@
+#Slicing Lab: Exercise 3.1
+
+a_string = "this is a string"
+a_tuple = (2, 54, 13, 12, 5, 32)
+
+print('Function 1: exchange_first_last(seq)')
+print('Function 2: every_other_removed(seq')
+print('Function 3: trim_four(seq)')
+print('Function 4: reverse(seq)')
+print('Function 5: third_shuffle(seq)')
+
+
+def exchange_first_last(seq):
+    if len(seq) == 1:
+        return seq
+    else:
+        return seq[-1:] + seq[1:len(seq)-1] + seq[:1]
+
+def every_other_removed(seq):
+    return seq[::2]
+
+def trim_four(seq):
+    return seq[4:len(seq)-4]
+
+def reverse(seq):
+    return seq[::-1]
+
+def third_shuffle(seq):
+    length = len(seq)
+    first_th = round(length/3)
+    last_th = round(2*length/3)
+    return seq[last_th:] + seq[:first_th] + seq[first_th:last_th]
```

Comment:
This is a fine solution - most I see are some variation of this.  Nice use of slicing in this exercise overall!